apiary.apib

```diff
@@ -1242,6 +1242,7 @@ Get a list of calendar events.
             + activity_type_id: `edd94120-63e2-4b10-80ed-fdfcfaa0a515` (string, optional)
             + ends_after: `2017-01-01T00:00:00+00:00` (string, optional) - Start of the period for which to return events
             + starts_before: `2018-01-01T00:00:00+00:00` (string, optional) - End of the period for which to return events
+            + term: `coffee` (string, optional) - Filters on title and description
```

Comment:
I think the description can be clearer, eg; `searches for a term in title or description`